### PR TITLE
Adding PyTest and github workflow

### LINKED ISSUE #52: Adding PyTest and github workflow

### PR DESCRIPTION
**Pull request by patrisiakaplun**
adding PyTest to pipfile.
adding pytest.yml file which create a github workflow to test code file of every PR
adding pytest.ini file and pipe file.lock file.
adding tests.py file

resolves #52

### CODE REVIEW

**Review by Yarboa — COMMENTED** [review]
I think you should apply  pipenv lock
And add pipefile.lock

Once sync is running, it is all installed
Please review first challenge

**Comment by patrisiakaplun** [comment]
@Yarboa pipefile.lock added.

File: pytest.ini [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[pytest]
```
Comment (guybeckenstein) [inline pytest.ini:1]: add this line
DJANGO_SETTINGS_MODULE = Pro4U.settings

Comment (patrisiakaplun) [reply pytest.ini:1]: @guybeckenstein done

**Review by guybeckenstein — APPROVED** [review]
LGTM

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**